kinto_amo/views/services.py

```diff
@@ -9,13 +12,58 @@
 
 @blocklist.get()
 def get_blocklist(request):
-    api_ver = request.matchdict['api_ver']
+    # api_ver = request.matchdict['api_ver']
     app = request.matchdict['application_guid']
-    app_ver = request.matchdict['application_ver']
-    query = request.body
-
-    return {
-        'API version': api_ver,
-        'Application': app,
-        'Application version': app_ver,
-        'Query': query}
+    # app_ver = request.matchdict['application_ver']
+
+    last_update = 0
+
+    # Addons blocklist
+    addons_records, addons_records_count = request.registry.storage.get_all(
+        collection_id="record",
+        parent_id="/buckets/staging/collections/add-ons")
```

Comment:
nit: maybe put bucket_id and collections in variable (at least before they become settings)